File: Examples/CSharp/WorkingWithCurrencies/GetCurrencySymbol.cs
namespace Aspose.Tasks.Examples.CSharp.WorkingWithCurrencies
{
    using System;

    internal class GetCurrencySymbol
    {
        public static void Run()
        {
            var dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod());

            //ExStart:GetCurrencySymbol
            //ExFor: Prj.CurrencySymbol
            //ExSummary: Shows how to get the project currency symbol.
            var project = new Project(dataDir + "project.mpp");
            Console.WriteLine(project.Get(Prj.CurrencySymbol));
            //ExEnd:GetCurrencySymbol
        }
    }
}

File: Examples/VisualBasic/WorkingWithCurrencies/GetCurrencySymbol.vb

Namespace WorkingWithCurrencies
    Public Class GetCurrencySymbol
        Public Shared Sub Run()
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType.FullName)

            ' ExStart:GetCurrencySymbol
            Dim project1 As Project = New Project(dataDir + "project.mpp")
            Console.WriteLine(project1.Get(Prj.CurrencySymbol))
            ' ExEnd:GetCurrencySymbol
        End Sub
    End Class
End Namespace
